ontology-assets/ontology_info.json

```diff
@@ -1,4 +1,98 @@
 {
+  "5.3.1-alpha": {
+    "ontologies": {
+      "CL": {
+        "version": "v2024-08-16",
+        "source": "https://github.com/obophenotype/cell-ontology/releases/download",
+        "filename": "cl.owl",
+        "cross_ontology_mapping": "cl.sssom.tsv"
+      },
+      "EFO": {
+        "version": "v3.74.0",
+        "source": "https://github.com/EBISPOT/efo/releases/download",
+        "filename": "efo.owl",
+        "additional_ontologies": [
+          "ZFS"
```

Comment:
was this meant to be CL?

Reply:
lol yes it was. i was wondering why no new ontology file was generated